<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagManagementWidget</class>
 <widget class="QWidget" name="TagManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Unknown Type タグ一覧</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetTags">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>選択</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tag</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Source Tag</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Type</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBottom">
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Status: Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerBottom">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUpdate">
       <property name="text">
        <string>更新実行</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
